lib/widgets/layouts/adaptive_layout/app_adaptive_scaffold_body_view.dart

```diff
@@ -28,8 +28,6 @@ class AppAdaptiveScaffoldBodyView extends StatelessWidget {
   final VoidCallback onOpenSettings;
   final AbsAppAdaptiveScaffoldBodyArgs? adaptiveScaffoldBodyArgs;
 
-  final ValueNotifier<String?> activeRoomIdNotifier;
```

Comment:
why we need to remove this?

Reply:
it was there just to be passed as parameter to _ColumnPageView. But I created a singleton which is used by the widgets who needs it

Reply:
https://github.com/linagora/twake-on-matrix/blob/4d9237da71f5720fa7b516558940f729a6fdba70/lib/presentation/notifiers/active_room_id_notifier.dart